java/client/src/main/java/glide/api/models/RedisValue.java

```diff
@@ -0,0 +1,36 @@
+package glide.api.models;
+
+import java.util.Map;
+import lombok.Getter;
+
+/**
+ * A union-like type which can store single or bulk value retrieved from Redis.
+ *
+ * @param <T> The wrapped data type
+ */
+@Getter
+public class RedisValue<T> {
```

Comment:
I'm not sure if we've discussed this already (I feel like we may have), but would ClusterValue be a better name? It seems we don't use this for standalone clients.

Reply:
Renamed, thanks